src/data-validation/schema/array-items/dataset-link.schema.json

```diff
@@ -0,0 +1,24 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "array-items/dataset-link.schema.json",
+    "title": "Dataset Link",
+    "description": "Schema for each single dataset",
+    "additionalProperties": false,
+    "type": "object",
+    "properties": {
+        "date": {
+            "type": "string"
```

Comment:
I can't remember what the date is supposed to mean here: is it a publication date or more like a file timestamp date?  I think you can put a description string here for documentation.

Reply:
I think a file timestamp data is associated with the key name - `dateCreated`, and the `date` here refers to the publication date. We can rename `date` to `dataReleased` to make it clear.